templates/taxi/index.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block title %}
+  <title>Information about our taxi's</title>

```

Comment:
The `title` block should not include the `<title>` tags, as these are already included in the `base.html` template. Instead, just provide the text for the title, like `Information about our taxi's`.